src/main/java/com/google/sps/servlets/OverTimeCasesServlet.java

```diff
@@ -0,0 +1,287 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Retrieves historical confirmed case data
+ * starting from 1-22-20. Takes coordinates
+ * in request and returns closest report
+ */
+@WebServlet("/timereport")
+public class OverTimeCasesServlet extends HttpServlet {
+  private Map<LocLatLng, List<Integer>> usTimeReports;
+  private Map<LocLatLng, List<Integer>> globalTimeReports;
+  private List<Integer> worldCases;
+  private List<String> dates;
+  public static final String CTYPE = "application/json"; // HttpServletResponse content type
+  public static final String ENCODING = "UTF-8"; // HttpServletResponse character encoding
```

Comment:
Are these referenced outside of the class? If not, it's better to define as private.
Alternatively, you can refactor to create a constants.java file to place all constants, including https://github.com/googleinterns/step168-2020/blob/master/src/main/java/com/google/sps/servlets/LinkServlet.java#L38-L39 and others.